src/Doctrine/DBAL/Types/PhoneNumberType.php

```diff
@@ -41,6 +41,7 @@ public function getSQLDeclaration(array $column, AbstractPlatform $platform): st
     {
         // DBAL < 4
         if (method_exists(AbstractPlatform::class, 'getVarcharTypeDeclarationSQL')) {
+            // @phpstan-ignore-next-line
```

Comment:
@maxhelias , it would be better to add the specific error in the phpstan file!

Reply:
Oh, I didn't know about this new feature. We'll edit next time.
Thanks